nordicsemi/__main__.py

```diff
@@ -537,12 +537,18 @@ def pkg():
                    '\n|s112_nrf52_6.1.1|0xB8|'
                    '\n|s112_nrf52_7.0.0|0xC4|'
                    '\n|s112_nrf52_7.0.1|0xCD|'
+                   '\n|s112_nrf52_v7.2.0|0x0103|'
```

Comment:
Please check whether it's correct to have this `v` before the version, since other lines don't have it.